### ISSUE TITLE
Song Automatically Skipping 

### ISSUE THREAD

**Issue by cosmic-lamanite** [hrkfdn/ncspot#883]
**Describe the bug**

It seems as though it is skipping my songs, as if its unavailable, but it does this with every song so its an everlasting skip, trying to force loop a song just makes it stuck
**To Reproduce**
1. Open Ncspot with your account
2. Click on a playlist or a song
3. It will start to skip all songs non stop

**Expected behavior**
I expect to play music 

**System (please complete the following information):**
 - OS: Linux
 - Terminal: Yakuake
 - Version: the latest, 10.0.0
 - Installed from: AUR

**Backtrace/Debug log**

thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: SendError((178, [0, 8, 0, 0, 0, 0, 0, 0, 0, 53, 1, 0, 1, 0, 61, 10, 54, 104, 109, 58, 47, 47, 109, 101, 116, 97, 100, 97, 116, 97, 47, 51, 47, 116, 114, 97, 99, 107, 47, 101, 100, 50, 101, 51, 49, 98, 53, 102, 51, 102, 50, 52, 52, 56, 100, 98, 101, 98, 51, 98, 55, 56, 101, 98, 101, 101, 52, 101, 97, 54, 56, 26, 3, 71, 69, 84]))', /home/pi/.cargo/registry/src/github.com-1ecc6299db9ec823/librespot-core-0.4.1/src/session.rs:213:48
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

**Additional context**
Im sorry if i can't really give a good description on what is happening, i tried to clear my credentials, wipe cache, and rebooted my computer and nothing

**Comment by Bettehem** [comment]
This is a duplicate of #864

**Comment by cosmic-lamanite** [comment]
> This is a duplicate of #864

Okay with this i found out how to fix this however not entirely.
i found out that by typeing [https://apresolve.spotify.com/](url) on my browser i would get the following string 
`{"ap_list":["ap-gue1.spotify.com:4070","ap-gue1.spotify.com:443","ap-gue1.spotify.com:80","ap-guc3.spotify.com:4070","ap-gew1.spotify.com:443","ap-gae2.spotify.com:80"]}`

with this i took the IP Address of `ap-gae2.spotify.com` by typing on terminal 
`ping ap-gae2.spotify.com` which will return `PING ap-gae2.spotify.com (104.199.240.237) 56(84) bytes of data.
64 bytes from 237.240.199.104.bc.googleusercontent.com (104.199.240.237): icmp_seq=1 ttl=104 time=306 ms` 
the Address IP is `104.199.240.237` so then i go to `/etc/hosts` and type the first hostname which for me is `ap-gue1.spotify.com` and the IP Address i have obtained.

in Conclusion, `/etc/hosts` should look like this for me:
`104.199.240.237 ap-gue1.spotify.com` 
and it worked!
